ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml

```diff
@@ -282,6 +282,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Set here to false

Reply:
Done 

https://github.com/openshift/release/compare/8839a1347c13bbd230f829c05c58049a4cef794b..24b3cf4beaec4b1851ac1dbdeac997872a2ed225